### DISCUSSION TITLE
Just wanted to thank the dev(s) for building this!

### DISCUSSION THREAD

**Discussion by inform880** [mrlt8/docker-wyze-bridge#517] [General]
I'm personally a mobile dev working in React Native (TS) and I (mostly) understand how much work this must be to maintain, especially for so many cameras. Thank you! 

If you need any mobile work done, let me know!

**Comment by mrlt8** [comment]
Thanks, help and ideas are always appreciated! 
A mobile app is probably overkill, but the WebUI could probably benefit if we moved to a JS framework/library like react or vue as we add more features to control the the cameras.

**Comment by JA16122000** [comment]
I'm new in this form of using programs like this.
Special in docker-compose!

I'm using now a raspi for a few years, but thanks to this bridge i found by google, i'm in to docker-compose!

I love this Wyze-Bridge and it works like a charm! :D
Thanks for al your hard work en perfect writing, so everthing will work like it should ;)

PS. Sorry for my bad English, i'm from the Netherlands
